CalDAVClient.php

```diff
@@ -174,7 +174,7 @@ function getetags() {
     
     // get the ctag of the calendar on the server
     // @see https://sabre.io/dav/building-a-caldav-client/
-    function getctag() {
+    function getCTag() {
```

Comment:
IIUC this renaming should also be performed on:
- `FSAgenda.php` (L126, and similarly on L135)
- `tests/CalDAVTest.php`
- `tests/FilesystemCacheTest.php`